2024-02-11 12:38:56,019 DEBUG 4XufSm5mhT : POST /login/customer [in user.py: 64]
2024-02-11 12:38:56,165 DEBUG 4XufSm5mhT : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 12:38:56,165 DEBUG 4XufSm5mhT : AuthDAO: tried to fetch user with email diy@gmail.com in table cust_auth [in authdao.py: 34]
2024-02-11 12:38:56,165 INFO 4XufSm5mhT : Customer with c_id:FRnZq logged in [in customer_login_handler.py: 33]
2024-02-11 12:38:56,165 INFO 4XufSm5mhT : Customer FRnZq logged in. [in customer_login_controller.py: 19]
2024-02-11 18:03:09,947 ERROR 9KG2MfjDJK : Employee Login: Invalid email kelly@manager.com provided. [in employee_login_handler.py: 22]
2024-02-11 18:04:56,343 ERROR 5ue3WoEvSC : Error while adding feedback for ticket Np6HU. Error 1136 (21S01): Column count doesn't match value count at row 1 [in feedback_handler.py: 44]
2024-02-11 18:07:05,612 ERROR 5wvgCFkxZp : Error while adding feedback for ticket Np6HU. Error 1136 (21S01): Column count doesn't match value count at row 1 [in feedback_handler.py: 44]
2024-02-11 23:01:38,323 DEBUG G9zWJSApuU : POST /login/employee [in user.py: 77]
2024-02-11 23:01:38,612 DEBUG G9zWJSApuU : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 23:01:38,614 DEBUG G9zWJSApuU : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-11 23:01:38,616 INFO G9zWJSApuU : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-11 23:01:38,619 INFO G9zWJSApuU : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-11 23:02:18,192 DEBUG CJsysZNTcZ : POST /login/employee [in user.py: 77]
2024-02-11 23:02:18,204 DEBUG CJsysZNTcZ : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 23:02:18,205 DEBUG CJsysZNTcZ : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-11 23:02:18,206 INFO CJsysZNTcZ : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-11 23:02:18,207 INFO CJsysZNTcZ : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-11 23:04:30,234 DEBUG FiJB5SDHLk : POST /login/employee [in user.py: 77]
2024-02-11 23:04:30,245 DEBUG FiJB5SDHLk : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 23:04:30,246 DEBUG FiJB5SDHLk : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-11 23:04:30,246 INFO FiJB5SDHLk : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-11 23:04:30,247 INFO FiJB5SDHLk : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-12 10:45:59,693 DEBUG DQZGCt4nBG : POST /login/employee [in user.py: 77]
2024-02-12 10:45:59,822 DEBUG DQZGCt4nBG : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-12 10:45:59,823 DEBUG DQZGCt4nBG : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-12 10:45:59,824 INFO DQZGCt4nBG : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-12 10:45:59,825 INFO DQZGCt4nBG : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-15 15:01:17,031 DEBUG 8JJMgBiLjG : POST /login/customer [in user.py: 65]
2024-02-15 15:01:17,125 DEBUG 8JJMgBiLjG : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:01:17,125 DEBUG 8JJMgBiLjG : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:01:17,125 INFO 8JJMgBiLjG : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:01:17,125 INFO 8JJMgBiLjG : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:07:48,286 DEBUG 78TJKnS7MS : POST /login/customer [in user.py: 65]
2024-02-15 15:07:48,286 DEBUG 78TJKnS7MS : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:07:48,286 DEBUG 78TJKnS7MS : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:07:48,286 INFO 78TJKnS7MS : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:07:48,302 INFO 78TJKnS7MS : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:35:44,301 DEBUG 3pCQDzitLP : POST /login/customer [in user.py: 65]
2024-02-15 15:35:44,316 DEBUG 3pCQDzitLP : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:35:44,316 DEBUG 3pCQDzitLP : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:35:44,316 INFO 3pCQDzitLP : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:35:44,316 INFO 3pCQDzitLP : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:36:30,054 DEBUG t42jhwDdMZ : POST /login/customer [in user.py: 65]
2024-02-15 15:36:30,066 DEBUG t42jhwDdMZ : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:36:30,067 DEBUG t42jhwDdMZ : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:36:30,068 INFO t42jhwDdMZ : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:36:30,069 INFO t42jhwDdMZ : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:39:19,230 DEBUG JWDcKNu4uQ : POST /login/customer [in user.py: 65]
2024-02-15 15:39:19,246 DEBUG JWDcKNu4uQ : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:39:19,246 DEBUG JWDcKNu4uQ : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:39:19,246 INFO JWDcKNu4uQ : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:39:19,246 INFO JWDcKNu4uQ : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:40:16,512 DEBUG 8Tmc9DhFmh : POST /login/customer [in user.py: 65]
2024-02-15 15:40:16,533 DEBUG 8Tmc9DhFmh : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:40:16,533 DEBUG 8Tmc9DhFmh : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:40:16,533 INFO 8Tmc9DhFmh : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:40:16,533 INFO 8Tmc9DhFmh : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:41:17,379 DEBUG 3BUcKcewzt : POST /login/customer [in user.py: 65]
2024-02-15 15:41:17,395 DEBUG 3BUcKcewzt : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:41:17,395 DEBUG 3BUcKcewzt : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:41:17,395 INFO 3BUcKcewzt : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:41:17,395 INFO 3BUcKcewzt : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:44:18,110 DEBUG CdcBToWrvw : POST /login/customer [in user.py: 65]
2024-02-15 15:44:18,131 DEBUG CdcBToWrvw : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:44:18,131 DEBUG CdcBToWrvw : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:44:18,131 INFO CdcBToWrvw : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:44:18,131 INFO CdcBToWrvw : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:50:13,353 DEBUG FegC9WmDqC : POST /login/customer [in user.py: 65]
2024-02-15 15:50:13,432 DEBUG FegC9WmDqC : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:50:13,432 DEBUG FegC9WmDqC : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:50:13,432 INFO FegC9WmDqC : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:50:13,432 INFO FegC9WmDqC : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:51:05,430 DEBUG 7v5sT9GBUb : POST /login/customer [in user.py: 65]
2024-02-15 15:51:05,443 DEBUG 7v5sT9GBUb : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:51:05,444 DEBUG 7v5sT9GBUb : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:51:05,444 INFO 7v5sT9GBUb : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:51:05,445 INFO 7v5sT9GBUb : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:51:27,823 DEBUG KyVtA9iUjs : POST /login/customer [in user.py: 65]
2024-02-15 15:51:27,835 DEBUG KyVtA9iUjs : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:51:27,836 DEBUG KyVtA9iUjs : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:51:27,836 INFO KyVtA9iUjs : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:51:27,837 INFO KyVtA9iUjs : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:52:27,363 DEBUG J27JyC7KCv : POST /login/customer [in user.py: 65]
2024-02-15 15:52:27,385 DEBUG J27JyC7KCv : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:52:27,387 DEBUG J27JyC7KCv : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:52:27,388 INFO J27JyC7KCv : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:52:27,389 INFO J27JyC7KCv : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:55:01,339 DEBUG 5dTDTYGjZp : POST /login/customer [in user.py: 65]
2024-02-15 15:55:01,359 DEBUG 5dTDTYGjZp : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:55:01,360 DEBUG 5dTDTYGjZp : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:55:01,360 INFO 5dTDTYGjZp : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:55:01,361 INFO 5dTDTYGjZp : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:55:47,018 DEBUG JvHL7c8von : POST /login/customer [in user.py: 65]
2024-02-15 15:55:47,029 DEBUG JvHL7c8von : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:55:47,030 DEBUG JvHL7c8von : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:55:47,031 INFO JvHL7c8von : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:55:47,032 INFO JvHL7c8von : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:56:29,358 DEBUG EAEB38sDip : POST /login/customer [in user.py: 65]
2024-02-15 15:56:29,379 DEBUG EAEB38sDip : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:56:29,380 DEBUG EAEB38sDip : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:56:29,380 INFO EAEB38sDip : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:56:29,381 INFO EAEB38sDip : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:57:02,263 DEBUG DzYCrCrcew : POST /login/customer [in user.py: 65]
2024-02-15 15:57:02,275 DEBUG DzYCrCrcew : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:57:02,275 DEBUG DzYCrCrcew : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:57:02,276 INFO DzYCrCrcew : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:57:02,277 INFO DzYCrCrcew : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:57:29,780 DEBUG 73NGTqci8q : POST /login/customer [in user.py: 65]
2024-02-15 15:57:29,795 DEBUG 73NGTqci8q : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 15:57:29,795 DEBUG 73NGTqci8q : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 15:57:29,795 INFO 73NGTqci8q : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 15:57:29,795 INFO 73NGTqci8q : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 15:57:33,873 DEBUG BNurVwVZdy : POST /tickets [in ticket.py: 39]
2024-02-15 15:57:33,873 DEBUG BNurVwVZdy : DepartmentDAO: department with 1 fetched. [in department_dao.py: 16]
2024-02-15 15:57:33,873 ERROR BNurVwVZdy : Create Ticket: Database error 1406 (22001): Data too long for column 'title' at row 1 while creating new ticket. [in new_ticket_handler.py: 33]
2024-02-15 15:58:17,635 DEBUG 9rZ74h8zZy : POST /tickets [in ticket.py: 39]
2024-02-15 15:58:17,651 DEBUG 9rZ74h8zZy : DepartmentDAO: department with 1 fetched. [in department_dao.py: 16]
2024-02-15 15:58:17,662 INFO 9rZ74h8zZy : New ticket raised with ticket_id:VdV2C, title:Need Help. by customer:RPfe5 [in ticket_dao.py: 17]
2024-02-15 15:58:17,666 INFO 9rZ74h8zZy : Customer RPfe5 created new ticket VdV2C [in new_ticket_controller.py: 23]
2024-02-15 16:00:38,188 DEBUG Ep3qoxYkWi : POST /tickets [in ticket.py: 39]
2024-02-15 16:00:38,255 DEBUG Ep3qoxYkWi : DepartmentDAO: department with 1 fetched. [in department_dao.py: 16]
2024-02-15 16:00:38,255 INFO Ep3qoxYkWi : New ticket raised with ticket_id:FUpoN, title:Need Help. by customer:RPfe5 [in ticket_dao.py: 17]
2024-02-15 16:00:38,266 INFO Ep3qoxYkWi : Customer RPfe5 created new ticket FUpoN [in new_ticket_controller.py: 23]
2024-02-15 16:01:10,831 DEBUG A2mgzsqQdj : POST /login/customer [in user.py: 65]
2024-02-15 16:01:10,853 DEBUG A2mgzsqQdj : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 16:01:10,853 DEBUG A2mgzsqQdj : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 16:01:10,853 INFO A2mgzsqQdj : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 16:01:10,853 INFO A2mgzsqQdj : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 16:01:10,957 DEBUG JDjApHwFT3 : POST /tickets [in ticket.py: 39]
2024-02-15 16:01:10,957 DEBUG JDjApHwFT3 : DepartmentDAO: department with 1 fetched. [in department_dao.py: 16]
2024-02-15 16:01:10,957 INFO JDjApHwFT3 : New ticket raised with ticket_id:FjLyE, title:Need Help. by customer:RPfe5 [in ticket_dao.py: 17]
2024-02-15 16:01:10,973 INFO JDjApHwFT3 : Customer RPfe5 created new ticket FjLyE [in new_ticket_controller.py: 23]
2024-02-15 16:05:02,246 DEBUG BQ5Q92CYYo : POST /login/customer [in user.py: 65]
2024-02-15 16:05:02,325 DEBUG BQ5Q92CYYo : AuthDAO: customer auth and employee auth table created [in auth_dao.py: 26]
2024-02-15 16:05:02,325 DEBUG BQ5Q92CYYo : AuthDAO: tried to fetch user with email abc@gmail.com in table cust_auth [in auth_dao.py: 34]
2024-02-15 16:05:02,325 INFO BQ5Q92CYYo : Customer with c_id:RPfe5 logged in [in customer_login_handler.py: 33]
2024-02-15 16:05:02,325 INFO BQ5Q92CYYo : Customer RPfe5 logged in. [in customer_login_controller.py: 19]
2024-02-15 16:05:02,418 DEBUG 9is28vxkPs : POST /tickets [in ticket.py: 39]
2024-02-15 16:05:02,418 DEBUG 9is28vxkPs : DepartmentDAO: department with 1 fetched. [in department_dao.py: 16]
2024-02-15 16:05:02,418 INFO 9is28vxkPs : New ticket raised with ticket_id:S7KPn, title:Need Help. by customer:RPfe5 [in ticket_dao.py: 17]
2024-02-15 16:05:02,434 INFO 9is28vxkPs : Customer RPfe5 created new ticket S7KPn [in new_ticket_controller.py: 23]
2024-02-15 16:05:02,528 DEBUG 7CHkboUfEs : GET /tickets/{{new-ticket-id}} [in ticket.py: 54]
